python/doc/examples/functional_modeling/field_functions/plot_process_sample.py

```diff
@@ -0,0 +1,161 @@
+"""
+Create a process sample from a sample
+=====================================
+"""
+
+# %%
+# In this example, a :class:`~openturns.ProcessSample` is created from a sample.
```

Comment:
is created from data

as maybe sample is ambiguous in this context

Reply:
Thank you, done